### PR TITLE
chore: remove mentions of decommissioned Cloudflare hosts

### PR DESCRIPTION
**Pull request by lidel**
This PR removes mentions of `https://cf-ipfs.com` or `https://cloudflare-ipfs.com` and ensures content makes sense after August 14th, 2024

Context:
https://blog.cloudflare.com/cloudflares-public-ipfs-gateways-and-supporting-interplanetary-shipyard

Supersedes / Closes #1865 + **fixes bunch of broken links that CI identified**.

### CODE REVIEW

File: docs/how-to/gateway-best-practices.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ If you are running an IPFS node that is also configured as an IPFS gateway, each
 - Put a CDN like Cloudflare in front of the IPFS gateway.
 - Consider enabling the [Accelerated DHT Client](https://github.com/ipfs/go-ipfs/blob/master/docs/experimental-features.md#accelerated-dht-client). 
 - Test and monitor your internet connection speed, with a tool like [Speedtest CLI](https://www.speedtest.net/apps/cli).
-- Monitor disk I/O and make sure that no other processes are causing disk I/O bottlenecks with a tool like [iotop](https://linux.die.net/man/1/iotop) or [iostat](https://linux.die.net/man/1/iostat).
```
Comment (aschmahmann) [inline docs/how-to/gateway-best-practices.md:38]: Why did these links change and why is one to GitHub and the other not (looks like iostat might be here https://github.com/sysstat/sysstat)?

**Review by aschmahmann — APPROVED** [review]
LGTM, but left a couple comments

File: docs/how-to/gateway-best-practices.md (hunk 1 above)
Comment (lidel) [reply docs/how-to/gateway-best-practices.md:38]: `linux.die.net`  return 403 to requests coming from CI, likely some overzealous abuse protection. I'll switch the iostat to github too, thanks!

File: docs/how-to/gateway-best-practices.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ If you are running an IPFS node that is also configured as an IPFS gateway, each
...
+- Monitor disk I/O and make sure that no other processes are causing disk I/O bottlenecks with a tool like [iotop](https://github.com/Tomas-M/iotop) or [iostat](https://en.wikipedia.org/wiki/Iostat).
```
Comment (lidel) [inline docs/how-to/gateway-best-practices.md:38]:
```suggestion
- Monitor disk I/O and make sure that no other processes are causing disk I/O bottlenecks with a tool like [iotop](https://github.com/Tomas-M/iotop) or [iostat](https://github.com/sysstat/sysstat).
```

**Pull request merged**